pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -153,18 +153,21 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, el *v1alpha1.EventListene
 	return nil
 }
 
-func reconcileObjectMeta(oldMeta *metav1.ObjectMeta, newMeta metav1.ObjectMeta) (updated bool) {
-	if !reflect.DeepEqual(oldMeta.Labels, newMeta.Labels) {
+func reconcileObjectMeta(existing *metav1.ObjectMeta, desired metav1.ObjectMeta) (updated bool) {
+	if !reflect.DeepEqual(existing.Labels, desired.Labels) {
 		updated = true
-		oldMeta.Labels = newMeta.Labels
+		existing.Labels = desired.Labels
 	}
-	if !reflect.DeepEqual(oldMeta.Annotations, newMeta.Annotations) {
+
+	// TODO(dibyom): We should propagation of some labels such as `kubernetes.io/last-applied-revision`
```

Comment:
```suggestion
	// TODO(dibyom): We should propagation of some annotations such as `kubernetes.io/last-applied-revision`
```
is that annotations or labels ?

Reply:
done!